services/order-service/Dockerfile

```
@@ -0,0 +1,16 @@
+# Stage 1: Build
+FROM maven:3.9-eclipse-temurin-17 AS build
+WORKDIR /app
+COPY pom.xml .
+COPY services/order-service/pom.xml services/order-service/
+COPY services/order-service/src services/order-service/src
+COPY shared shared
+RUN mvn -B -pl services/order-service -am install -DskipTests
+
+# Stage 2: Runtime
+FROM gcr.io/distroless/java17-debian12:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)